modules/ROOT/pages/clustering/server-syntax.adoc

```diff
@@ -0,0 +1,233 @@
+:description: This section provides a summary of server management Cypher syntax.
+[role=enterprise-edition]
+[[server-commands-reference]]
+= Server commands reference
```

Comment:
It's too long for SEO-purposes, that's why I shortened it. Do you have a shorter suggestion?

Reply:
I see - I would be inclined to just call it "Server commands syntax" as opposed to "reference" ´, then, but either works. What do you think, @renetapopova?

Reply:
Since the previous section is called _Settings reference_, I wanted to follow that for consistency, since it is a referential section. The syntax is explained elsewhere.